.github/workflows/test-ui.yml

```diff
@@ -47,12 +47,29 @@ jobs:
         run: cd ui && yarn run lint:js
       - name: lint:hbs
         run: cd ui && yarn run lint:hbs
-      - name: Ember tests
+  tests:
+    needs:
+      - lint
+    strategy:
+      matrix:
+        partition: [1, 2, 3, 4]
+        split: [4]
```

Comment:
I want there to be a `len` function of some kind available, instead of this `[4]`, but I'm not finding one, so fine. :yum: